templates/default/systemd.erb

```diff
@@ -4,10 +4,11 @@ After=network.target
 Wants=network.target
 
 [Service]
+EnvironmentFile=-/etc/sysconfig/<%= @name %>
```

Comment:
the service yes. but don't need to reload systemctl itself.
but for sure I also can change this.